core/src/test/java/com/google/errorprone/bugpatterns/AlwaysThrowsTest.java

```diff
@@ -42,6 +42,21 @@ public void positive() {
         .doTest();
   }
 
+  @Test
+  public void positiveUUID() {
+    testHelper
+        .addSourceLines(
+            "Test.java",
+            "import java.util.UUID;",
+            "class T { ",
+            "  void f() {",
+            "    // BUG: Diagnostic contains: will fail at runtime with a IllegalArgumentException",
```

Comment:
should this be "an IllegalArgumentException"?

Reply:
I suppose it should, we could consider updating [the logic that constructs the diagnostic](https://github.com/google/error-prone/pull/3012/files#diff-870a751c3cdcc95fa86d5c4bcceba95518f3d400c7752641785735b03a74bbb1R189) to check for vowels as a follow-up